### PR TITLE
test: pull testbench docker image, kokoro presubmit conformance test …

### PR DESCRIPTION
**Pull request by ddelgrosso1**
…hooks

Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/nodejs-storage/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

**Comment by ddelgrosso1** [comment]
There is currently a `conformance-test.yaml` that appears to run the tests on every pull request. However, it seems like most other tests / workflows use the .cfg / .sh approach. Not sure which is better and if we should just remove the older yaml file?

**Comment by shaffeeullah** [comment]
This looks good to me, but you should also request review from someone who actually knows how this is supposed to work.

File: conformance-test/testBenchUtil.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+export function runTestBenchDockerImage(): Promise<Buffer> {
+  return new Promise((resolve, reject) => {
+    try {
+      resolve(execSync(RUN_CMD));
+    } catch (err) {
+      reject(err);
+    }
+  });
+}
```
Comment (danielbankhead) [inline conformance-test/testBenchUtil.ts:46]: [Optional] This block can be streamlined to:

```suggestion
export async function getTestBenchDockerImage(): Promise<Buffer> {
  return execSync(PULL_CMD);
}

export async function runTestBenchDockerImage(): Promise<Buffer> {
  return execSync(RUN_CMD);
}
```

Note `execSync` would still blocking the main thread - if you're interested in having this more non-blocking, [`exec`](https://nodejs.org/api/child_process.html#child_process_child_process_exec_command_options_callback) might be a good choice

**Review by danielbankhead — COMMENTED** [review]
The TS looks good; not familiar enough with the Kokoro components yet to make a call

File: .kokoro/presubmit/node10/conformance-test.cfg [hunk 2]
```diff
@@ -0,0 +1,4 @@
+env_vars: {
```
Comment (fhinkel) [inline .kokoro/presubmit/node10/conformance-test.cfg:1]: If you look at the internal job configs, you see that we only have node12 configured. This file should be here: https://github.com/googleapis/nodejs-storage/tree/main/.kokoro/presubmit/node12

File: conformance-test/testBenchUtil.ts [hunk 3]
```diff
@@ -0,0 +1,34 @@
+/*!
```
Comment (bcoe) [inline conformance-test/testBenchUtil.ts:1]: nit: I would be tempted to cll this `test-bench-util.ts`, this just tends to be the JavaScript convention.

**Review by bcoe — APPROVED** [review]
Nits about filenames aside, LGTM.

**Pull request merged**